src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -226,6 +226,23 @@ public static String desanitizeFromHtml(String sanitizedString) {
     }
 
     /**
+     * Returns true if html contains substrings of sanitized characters
+     */
+    public static boolean isSanitizedHtml(String html) {
+        if (html == null) {
+            return false;
+        }
+        if (html.indexOf('<') >= 0 || html.indexOf('>') >= 0 || html.indexOf('\"') >= 0
+                || html.indexOf('/') >= 0 || html.indexOf('\'') >= 0) {
+            return false;
```

Comment:
The editor in browser does sanitization for what is entered in it but would still produce tags like "<p>" . So this check is necessary to avoid false positives.